src/searcher.rs

```diff
@@ -233,6 +235,43 @@ impl Searcher {
         })
     }
 
+    #[pyo3(signature = (search_query, agg_query))]
+    fn aggregate(
+        &self,
+        py: Python,
+        search_query: &Query,
+        agg_query: String,
```

Comment:
A second point is for naming.  Here you have the `search_query` parameter, but in the `search()` function around line 167, the parameter is just called `query`. I think we should keep it the same here for consistency.

Then in the underlying `AggregationCollector::from_aggs()` function, the parameter with the aggregations is called `agg`. I think we should make it the same here, again for consistency against the underlying call.

Reply:
Good point! I've renamed these parameters